package.json

```diff
@@ -33,7 +33,7 @@
     "test-parallel": "rm nonce.db ; rm sudo.lock; jest test/parallel/* --verbose --ci",
     "test-playground-pararell": "rm nonce.db ; rm sudo.lock; jest test/exploratory/parallelTesting/transfer/* --verbose",
     "test-sequential": "rm nonce.db ; rm sudo.lock; jest test/sequential/* --verbose --runInBand",
-    "test-sequential-e2e": "rm nonce.db ; rm sudo.lock; jest test/sequential/xyk-pallet.test.ts --verbose --runInBand --ci ",
+    "test-sequential-e2e": "rm nonce.db ; rm sudo.lock; jest test/sequential/* --verbose --runInBand --ci ",
     "test-ci": "rm nonce.db ; rm sudo.lock; jest test/ci/* --verbose --runInBand --ci ",
```

Comment:
@mateuszaaa , this will add more coverage, but test run will take 20min,
ping me if thats ok for you :)